docs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6-slim
+FROM python:3.7-slim
```

Comment:
This one was still using 3.6. Caching-wise, it's easier if all the containers are using the same base image.